plugins/node/opentelemetry-instrumentation-mongodb/package.json

```diff
@@ -59,7 +60,7 @@
     "codecov": "3.8.3",
     "gts": "3.1.0",
     "mocha": "7.2.0",
-    "mongodb": "3.6.11",
+    "mongodb": "4.2.2",
```

Comment:
We are installing mongodb v4 as dev dependency, which means that v3 tests will no pass, only in tav, right?

Reply:
Right :)

Reply:
What do you think about documenting it somehow?
Maybe we can write a comment at the top of the v3 tests file with instructions on how to run them locally.
The next person who will try to run v3 test will see all tests failing and be surprised

Reply:
Added doc :)